src/libImaging/JpegEncode.c

```diff
@@ -278,7 +278,7 @@ ImagingJpegEncode(Imaging im, ImagingCodecState state, UINT8 *buf, int bytes) {
 
         case 4:
 
-            if (context->comment_size > 0) {
+            if (context->comment) {
```

Comment:
the rest of the code is conditional on `comment != NULL` rather than `comment_size > 0`

they should be the same, but keeping things consistent feels nicer to me

Reply:
When you say "the rest of the code", what are you referring to?